Sources/AWSSDKSwiftCore/String.swift

```diff
@@ -6,7 +6,42 @@
 //
 //
 
-import Foundation
+let swiftReservedWords: [String] = [
+    "protocol",
+    "return",
+    "operator",
+    "class",
+    "struct",
+    "break",
+    "continue",
+    "extension",
+    "self",
+    "public",
+    "private",
+    "internal",
+    "where",
+    "catch",
+    "try",
+    "default",
+    "case",
+    "static",
+    "switch",
+    "if",
+    "else",
+    "func",
+    "enum",
+    "true",
+    "false",
+    "nil",
+    "in",
+    "import",
+    "as",
+    "is",
+    "do",
+    "try",
+    "type",
+    "repeat"
+]
```

Comment:
and why is this here?

Reply:
cherry pick. damn it!

Reply:
fixed.

Reply:
I'm not a keen fan of cherry pick for this very reason